src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -58,32 +59,30 @@
             :justify-content :center}
    [icon/icon icon {:color colors/white}]])
 
-(defn- activity-unread-dot
-  []
-  [rn/view {:margin-left      14
-            :margin-right     6
-            :background-color colors/primary-50
-            :width            8
-            :height           8
-            :border-radius    4}])
-
 (defn- activity-context
   [context replying?]
-  (let [first-line-offset (if replying? 4 -2)
+  (let [first-line-offset (if replying? 4 0)
         gap-between-lines 4]
-    (into [rn/view {:flex-direction :row
-                    :align-items    :center
-                    :flex-wrap      :wrap
-                    :margin-top     first-line-offset}]
-          (map-indexed (fn [index detail]
-                         ^{:key index}
-                         [rn/view {:margin-right 4
-                                   :margin-top   gap-between-lines}
-                          (if (string? detail)
-                            [text/text {:size :paragraph-2}
-                             detail]
-                            detail)])
-                       context))))
+    (into [rn/view {:style {:flex-direction  :row
+                            :flex            1
+                            :align-items     :center
+                            :justify-content :flex-start
+                            :flex-wrap       :wrap
+                            :margin-top      first-line-offset}}]
+          (mapcat (fn [detail]
+                    ^{:key (hash detail)}
+                    (if (string? detail)
+                      (map-indexed (fn [index s]
+                                     ^{:key index}
+                                     [rn/view {:style {:margin-right 4
+                                                       :margin-top   0}}
+                                      [text/text {:size :paragraph-2}
+                                       s]])
+                                   (str/split detail #"\s+"))
+                      [[rn/view {:margin-right 4
```

Comment:
Extra vector maybe ?

Reply:
This is actually necessary because of the parent s-exp `mapcat`.

Reply:
Got it, Thanks for letting me know Icaro!